### PR TITLE
[CPU][RISC-V] Re-enable MobileNetV2 int8 benchmarks.

### LINKED ISSUE #16061: RISC-V failed to build int8 MobileNetV2 with LLVM Integrate #16055
To repro:

```
iree-compile --output-format=vm-bytecode --iree-hal-target-backends=llvm-cpu --iree-llvmcpu-target-triple=riscv32-pc-linux-gnu --iree-llvmcpu-target-cpu=generic-rv32 --iree-llvmcpu-target-abi=ilp32 --iree-llvmcpu-target-cpu-features=+m,+a,+f,+zvl512b,+zve32f --riscv-v-fixed-length-vector-lmul-max=8 --iree-vm-emit-polyglot-zip=true ~/mobilenet_v2_int8.mlir
```

It is broken by https://github.com/llvm/llvm-project/pull/76793

### PR DESCRIPTION
**Pull request by hanhanW**
It seems to be fixed by some upstream commits, so we no longer hit compilation errors.

Closes https://github.com/openxla/iree/issues/16061

### CODE REVIEW

**Comment by hanhanW** [comment]
That's hold this a bit, because it will need to be disabled after landing integrate. We can re-enable it soon.

**Review by dcaballe — APPROVED** [review]

**Pull request merged**
